advocacy_docs/supported-open-source/patroni/installing_patroni.mdx

```diff
@@ -7,28 +7,51 @@ tags:
   - Patroni
 ---
 
-Patroni packages are provided through the PGDG `apt` and `yum` repositories.
+EDB provides Patroni to customers via the `edb-patroni` package. This package provides Patroni itself and all its dependencies, so there is no need to install additional packages on the Postgres nodes.
 
-See [Platform Compatibility](https://www.enterprisedb.com/resources/platform-compatibility#epas) for the supported OS list (only `Linux x86-64 (amd64)` currently).
+Extra packages which no longer have to be installed separately are the following:
```

Comment:
```suggestion
As the dependencies no longer have to be installed separately, there is no need to install the python3-cdiff, python3-psutil, python3-psycopg2, python3-ydiff, python3-click, python3-click, python3-six, python3-dateutil, python3-prettytable, python3-pyyaml, python3-urllib3, python3-etcd, python3-dns or python3-certifi packages.
```

Reply:
Wouldn't we need to also remove the list that comes later?

Reply:
Yes, ah I see you've done that.